tests/benchmark/stateful/bloatnet/depth_benchmarks/depth_10.sol

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+contract WorstCaseERC20 {
+    // ERC20 State
+    mapping(address => uint256) public balanceOf;
+    mapping(address => mapping(address => uint256)) public allowance;
+    uint256 public totalSupply;
+
+    // Token metadata - returning constants to save gas
+    string public constant name = "WorstCase";
+    string public constant symbol = "WORST";
+    uint8 public constant decimals = 18;
+
+    constructor() {
+        // Mint total supply to deployer
+        totalSupply = 1_000_000_000 * 10 ** 18; // 1 billion tokens
+        balanceOf[msg.sender] = totalSupply;
+
+        // Set all mined storage slots to 1
+        assembly {
+            sstore(0xfeb1bc66963690bd7d902e86ccaf4e0fa1ea72277653d012a3fed288892770fc, 1)
```

Comment:
Does the address miner also put as a constraint that the preimages have this "nibble by nibble" overlap for the required depth?

Mostly asking because what matters for this to be deep branches is that the hashes of the storage slot numbers satisfy this, not the storage numbers themselves (i.e. the storage slots are hashed to define which is the storage trie address).

(Maybe I'm missing something else from the whole PR setup!)